src/main/java/seedu/duke/command/CommandListClients.java

```diff
@@ -1,18 +1,25 @@
 package seedu.duke.command;
 
+//@@author zoranabc201
 import seedu.duke.ClientList;
+import seedu.duke.CommandStructure;
```

Comment:
Perhaps you might want to consider adding specific final String?
Because by importing the entire CommandStructure still feels like wildcard import since unused parameters is in CommandStructure.

By importing specific final strings being used, it also brings clarity to the reader and your code can be significantly shorter as well (You can just type `START_INDEX` instead of `CommandStructure.START_INDEX`